onnxruntime/core/providers/cpu/math/cumsum.cc

```diff
@@ -48,11 +48,40 @@ void SumSlices(const Tensor& input, Tensor& output,
 }
 }  // namespace
 
+namespace cumsum_op {
+
+static Status GetAxis(const Tensor* axis_tensor, int64_t input_rank, int64_t& axis_out) {
+  if (!axis_tensor)
```

Comment:
It is required to check if this tensor is a nullptr because as per spec this input is optional, but it provides no default value for axis. Hence, axis tensor is quite necessary to proceed.